lib/miq_unicode.rb

```diff
@@ -0,0 +1,45 @@
+module ManageIQ
+  module UnicodeString
+    refine String do
+      def UnicodeToUtf8
+        dup.force_encoding("UTF-16LE").encode("UTF-8")
```

Comment:
should this be simplified to `dup. UnicodeToUtf8!`?

Reply:
Done.